repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/spring/CoreSpringProvider.java

```diff
@@ -163,31 +164,31 @@ public void initializeCoreContext(String configRoot,
             // I'm not sure how well this will work, maybe it will fix the problem.
             GenericBeanDefinition mBeanServer = new GenericBeanDefinition();
             mBeanServer.setBeanClass(MBeanServerFactoryBean.class);
-            mBeanServer.setAutowireCandidate(true);
-            mBeanServer.setLazyInit(false); //Make sure that it's not lazy!
```

Comment:
These are the default values, so I just removed the redundant lines. We depend on other defaults implicitly, so I did not see a reason to be explicit about these.